DIGNDB.App.SmitteStop.API/Attributes/UploadKeysAuthorizationAttribute.cs

```diff
@@ -14,12 +16,16 @@ public class UploadKeysAuthorizationAttribute : ActionFilterAttribute
         private readonly IJwtValidationService _jwtValidationService;
         private readonly IAnonymousTokenKeySource _anonymousTokenKeySource;
         private readonly ITokenVerifier _tokenVerifier;
+        private readonly AnonymousTokenKeyStoreConfiguration _config;
 
-        public UploadKeysAuthorizationAttribute(IJwtValidationService jwtValidationService, IAnonymousTokenKeySource anonymousTokenKeySource, ISeedStore seedStore)
+        public UploadKeysAuthorizationAttribute(IJwtValidationService jwtValidationService,
+            IAnonymousTokenKeySource anonymousTokenKeySource, ISeedStore seedStore,
+            IOptions<AnonymousTokenKeyStoreConfiguration> config)
         {
             _jwtValidationService = jwtValidationService;
             _anonymousTokenKeySource = anonymousTokenKeySource;
             _tokenVerifier = new TokenVerifier(seedStore);
+            _config = config.Value;
```

Comment:
Could maybe add an "at-" prefix or similar to these names to empahize that the config relates to anonymous tokens only? Not important though.